code/modules/reagents/reagent_containers/hypospray.dm

```diff
@@ -149,16 +149,21 @@
 
 /obj/item/reagent_containers/hypospray/autoinjector/attack_self(mob/user as mob)
 	if(is_open_container())
+		to_chat(user, SPAN_WARNING("\The [src] hasn't been secured yet! (Alt-Click to secure)"))
+		return
+	inject(user, user, TRUE)
```

Comment:
this should be on a timer (`do_after`), if its not already the case, so people don't accidentally inject themselves, since people are used to clicking their sprite.